files/bootstrap.sh

```diff
@@ -36,6 +39,12 @@ function print_help {
   echo "--use-max-pods Sets --max-pods for the kubelet when true. (default: true)"
 }
 
+function log {
```

Comment:
I avoided a strict concept of log levels, because I think we'll pretty much always want all the lines when we need to troubleshoot. I prefixed statements with `INFO`/`ERROR` etc. purely out of convention, and to make it obvious what the intent of the statement was. I changed every `echo` I could find in the script to use the new `log` function, and updated the statements accordingly.